static/index.html

```diff
@@ -4,8 +4,18 @@
     <title>Decentralized Renderer - Generic Templates</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@400&display=swap" rel="stylesheet" />
+    <style type="text/css">
+      *,
+      *::before,
+      *::after {
+        box-sizing: border-box;
+      }
+      * {
+        margin: 0;
+      }
+    </style>
   </head>
-  <body style="background-color: white">
+  <body>
```

Comment:
Do we need to move the background colour property into the `input.css`, or are we safe to not specify background colour here?

Reply:
i think its fine to not specify